### DISCUSSION TITLE
Can't Full Screen on Gnome 41/Fedora 35

### DISCUSSION THREAD

**Discussion by oprizal** [KDE/ghostwriter#687] [General]
Recently I found that I can't go to full screen on Gnome 41/Fedora 35. It's happen on both version available from Fedora default repos or build it directly from the Github repos. 

The app was blinking and can't responds on input when I change to full screen mode by clicking lower right button. The way to back to normal was by push `esc` button and the app back to window mode.

Is there anything I'm missing to install? Something like in dependencies?

**Comment by santiagocezar** [comment]
I've had this issue for quite some time, even in GNOME 40, it has something to do with Wayland. There was an issue regarding this but they're disabled now and I can't understand why (seriously, even if they don't plan on fixing stuff, they're still useful if someone that wants to make a pull request).

Funnily, it does work if you start the app with fullscreen already enabled. To do this you enable fullscreen, close using Alt+F4, and open it again

**Comment by ScreamingPigeon** [comment]
Yep, I'm facing a similar issue on Fedora 39, GNOME 45. Going to full screen turns the application window into a stuttering white screen.

**Comment by nicolastaffin** [comment]
Same problem in Ubuntu 24.04 Wayland. Going to fullscreen creates a glitch or an empty screen ( Ghostwriter 23.08.5). Xorg is ok but not Wayland.

**Comment by Sylphystia** [comment]
This is still happening on Ubuntu 24.04 for me, is there any solution?
